bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/StyledTextRenderer.java

```diff
@@ -180,6 +183,15 @@ public LineInfo(LineInfo info) {
 			}
 		}
 	}
+
+	private static class LineDrawInfo {
```

Comment:
I wonder if that should be a record (so implicitly final fields)

Reply:
Thanks, I'll try that. I learned about `record` just a few days ago (my excuse is that I'm not a Java programmer)

Reply:
done